esphome/components/sensor/__init__.py

```diff
@@ -684,8 +687,13 @@ async def heartbeat_filter_to_code(config, filter_id):
 @FILTER_REGISTRY.register("timeout", TimeoutFilter, TIMEOUT_SCHEMA)
```

Comment:
```suggestion
@FILTER_REGISTRY.register("timeout", Filter, TIMEOUT_SCHEMA)
```
Might be cleaner to follow the existing pattern like `sliding_window_moving_average_filter_to_code`. Register with just `Filter` and then create the appropriate one?

Reply:
087ed48dba